src/mahoji/commands/easter.ts

```diff
@@ -1,8 +1,32 @@
-import { Time } from 'e';
+import { Bank, LootTable } from 'oldschooljs';
 
 import { client } from '../..';
-import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { newChatHeadImage } from '../../lib/util/chatHeadImage';
 import { OSBMahojiCommand } from '../lib/util';
+import { mahojiUserSettingsUpdate } from '../mahojiSettings';
+
+const UniqueTable = new LootTable().add('Chickaxe').add('Leia');
+
+// Users get effectively 1 roll of this per 10 minutes
+const DeliverRewardTable = new LootTable()
+	.add(new LootTable(), 1, 15)
```

Comment:
15 / (15+19) chance to get no loot?
44% seems a little rough but that's fine.

Also, I'm pretty sure you can do
```ts
const DeliverRewardTable = new LootTable({ limit: 34 })
    .add(egg1)
    .add(egg2)
...
```
And then you don't need to make the empty `LootTable`, it will just roll 1 - 34 and if it's outside the range of the actual drops, it gives nothing.